website/package.json

```diff
@@ -25,15 +25,14 @@
     "@hashicorp/react-search": "^2.1.0",
     "@hashicorp/react-section-header": "2.0.2",
     "@hashicorp/react-subnav": "3.2.6",
-    "@hashicorp/react-text-and-content": "4.1.4",
     "@hashicorp/react-text-split": "0.4.0",
-    "@hashicorp/react-text-split-with-code": "0.2.0",
+    "@hashicorp/react-text-split-with-code": "^1.0.2-alpha.55",
     "@hashicorp/react-text-split-with-image": "1.4.0",
     "@hashicorp/react-text-split-with-logo-grid": "1.3.0",
     "@hashicorp/react-use-cases": "1.0.6",
     "@hashicorp/react-vertical-text-block-list": "2.0.3",
     "babel-plugin-import-glob-array": "0.2.0",
-    "next": "9.4.4",
+    "next": "^9.5.5",
```

Comment:
Upgrading `next` to `9.5.x` was necessary to pull in the latest `nextjs-scripts`, based on [release notes for `nextjs-scripts@13.0.0`](https://github.com/hashicorp/nextjs-scripts/releases/tag/13.0.0).